### PR TITLE
doc: correct ct config docs

### PR DESCRIPTION
**Pull request by Baroshem**
### Related Issues

closes #

### Short Description of the PR

Correction to Commercetools configuration docs

### Pull Request Checklist

- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

### CODE REVIEW

File: packages/core/docs/commercetools/configuration.md [hunk 1]
```diff
@@ -9,180 +9,69 @@ Commercetools configuration is located in two places:
...
+- `authHost` - link to Commercetools Authentication Server. It is used to request an access token from commercetools OAuth 2.0 service. To choose the nearest service, please visit [Commercetools hosts list](https://docs.commercetools.com/api/authorization)
+- `projectKey` - name of your Commercetools project, i.e. `my-awesome-vsf-project`
+- `clientId` - unique Commercetools Client ID. Visit [Commercetools documentation](https://docs.commercetools.com/tutorials/getting-started#creating-an-api-client) for more details about creating an API Client
+- `clientSecret` - Commercetools secret API key. Visit [Commercetools documentation](https://docs.commercetools.com/tutorials/getting-started#creating-an-api-client) for more details about creating an API Client
+- `scopes` - The scope constrains the endpoints to which a client has access, and whether a client has read or write access to an endpoint. Visit [Commercetools documentation](https://docs.commercetools.com/api/scopes#top) for more details about Scopes.
 
-#### `acceptLanguage`
+### `acceptLanguage`
 
 Commercetools supports querying localised fields via an array of accepted languages - `acceptLanguage`.
```
Comment (filrak) [inline packages/core/docs/commercetools/configuration.md:68]: This sentence is not very much human-readable. I know you can explain it better! :)

File: packages/core/docs/commercetools/configuration.md [hunk 2]
```diff
@@ -9,180 +9,69 @@ Commercetools configuration is located in two places:
...
-        'manage_products:<PROJECT_KEY>',
-        /* other scope rules */
-      ]
-    },
-    currency: 'USD',
-    country: 'US'
-  }
-}
-```
+- `useNuxtI18nConfig` - when this variable is set to true, `@vue-storefront/commercetools/nuxt` package will use i18n config object provided in nuxt.config.js. When set to false, i18n config should be declared directly inside this package configuration.
```
Comment (filrak) [inline packages/core/docs/commercetools/configuration.md:21]: A link to i18n config makes sense here

File: packages/core/docs/commercetools/configuration.md [hunk 3]
```diff
@@ -9,180 +9,69 @@ Commercetools configuration is located in two places:
...
+            /* other scope rules */
+          ]
+        },
+        currency: 'USD',
+        country: 'US'
+      }
+    }
+  }
+};
 ```
```
Comment (filrak) [inline packages/core/docs/commercetools/configuration.md:58]:
```suggestion
# `api`
```

File: packages/core/docs/commercetools/configuration.md [hunk 4]
```diff
@@ -9,180 +9,71 @@ Commercetools configuration is located in two places:
...
-  authHost: string;
-  projectKey: string;
-  clientId: string;
-  clientSecret: string;
-  scopes: string[];
-}
-```
-
-#### `countries`, `currencies`, `locales`
+- `useNuxtI18nConfig` - when this variable is set to true, `@vue-storefront/commercetools/nuxt` package will use i18n config object provided in nuxt.config.js. When set to false, `i18n` config should be declared directly inside this package configuration. You can read more about `i18n` config in Vue Storefront [here](../advanced/internationalization.md)
```
Comment (filrak) [inline packages/core/docs/commercetools/configuration.md:21]:
```suggestion
- `useNuxtI18nConfig` - when this variable is set to true, `@vue-storefront/commercetools/nuxt` package will use `i18n` config object provided in `nuxt.config.js`. When set to false, `i18n` config should be declared directly inside this package configuration. You can read more about `i18n` config in Vue Storefront [here](../advanced/internationalization.md)
```

File: packages/core/docs/commercetools/configuration.md [hunk 5]
```diff
@@ -9,180 +9,71 @@ Commercetools configuration is located in two places:
...
 
-Array of accepted items
+If you do not have this package installed locally, please run:
```
Comment (filrak) [inline packages/core/docs/commercetools/configuration.md:23]: this part is obsolete imho

**Review by filrak — CHANGES_REQUESTED** [review]

File: packages/core/docs/commercetools/configuration.md [hunk 6]
```diff
@@ -9,180 +9,71 @@ Commercetools configuration is located in two places:
...
+- `projectKey` - name of your Commercetools project, i.e. `my-awesome-vsf-project`
+- `clientId` - unique Commercetools Client ID. Visit [Commercetools documentation](https://docs.commercetools.com/tutorials/getting-started#creating-an-api-client) for more details about creating an API Client
+- `clientSecret` - Commercetools secret API key. Visit [Commercetools documentation](https://docs.commercetools.com/tutorials/getting-started#creating-an-api-client) for more details about creating an API Client
+- `scopes` - The scope constrains the endpoints to which a client has access, and whether a client has read or write access to an endpoint. Visit [Commercetools documentation](https://docs.commercetools.com/api/scopes#top) for more details about Scopes.
 
-#### `acceptLanguage`
+### `acceptLanguage`
 
-Commercetools supports querying localised fields via an array of accepted languages - `acceptLanguage`.
+List of languages the client is able to understand, and which locale variant is preferred.
```
Comment (filrak) [inline packages/core/docs/commercetools/configuration.md:70]: still it's not understandable ;p

File: packages/core/docs/commercetools/configuration.md (hunk 5 above)
Comment (Baroshem) [reply packages/core/docs/commercetools/configuration.md:23]: This was a recommendation from @andrzejewsky. If you think it is not needed, I can remove it as well :)

File: packages/core/docs/commercetools/configuration.md (hunk 6 above)
Comment (Baroshem) [reply packages/core/docs/commercetools/configuration.md:70]: This is a default field description from Commercetools :D But I will try to make it more human readable

Comment (andrzejewsky) [reply packages/core/docs/commercetools/configuration.md:70]: just put the link to CT with one sentence eg "the array of possible locales comemrcetools will use, more details here"

File: packages/core/docs/commercetools/configuration.md [hunk 7]
```diff
@@ -9,180 +9,71 @@ Commercetools configuration is located in two places:
...
-  authHost: string;
-  projectKey: string;
-  clientId: string;
-  clientSecret: string;
-  scopes: string[];
-}
-```
-
-#### `countries`, `currencies`, `locales`
+- `useNuxtI18nConfig` - when this variable is set to true, `@vue-storefront/commercetools/nuxt` package will use `i18n` config object provided in `nuxt.config.js`. When set to false, `i18n` config should be declared directly inside this package configuration. You can read more about `i18n` config in Vue Storefront [here](../advanced/internationalization.md)
```
Comment (filrak) [inline packages/core/docs/commercetools/configuration.md:21]:
```suggestion
- `useNuxtI18nConfig` - when this property is set to true, `@vue-storefront/commercetools/nuxt` package will use `i18n` config object provided in `nuxt.config.js`. When set to false, `i18n` config should be declared directly inside this package configuration. You can read more about `i18n` config in Vue Storefront [here](../advanced/internationalization.md)
```

File: packages/core/docs/commercetools/configuration.md (hunk 5 above)
Comment (filrak) [reply packages/core/docs/commercetools/configuration.md:23]: Please remove as this is not the only thing once should do to make vsf work and this usage scenario wasn'e even tested.

**Review by filrak — APPROVED** [review]
One small thing left to change (in a comment0, apart from that it's great. You can merge once you correct it ;)

**Pull request merged**